lib/scenic/statements.rb

```diff
@@ -89,6 +89,32 @@ def update_view(name, version: nil, revert_to_version: nil, materialized: false)
       end
     end
 
+    # Update a database view to a new version using `CREATE OR REPLACE VIEW`.
+    #
+    # The existing view is created/replaced using the supplied `version`
+    # parameter.
+    #
+    # Does not work with materialized views, no parameter for that is provided!
```

Comment:
What is the error I get if I call this with `materialized: true` on a materialized view? What is the error if I call it without the keyword but the view is materialized? I think it might be worth accepting that as a parameter and giving a nice, specific error if it'll help in these cases. What do you think?